test/starknet/shared/helpers.ts

```diff
@@ -17,3 +32,74 @@ export function bytesToHex(bytes: number[]): string {
   }).join('');
   return '0x' + body;
 }
+
+export async function setup() {
```

Comment:
do you think it makes sense to put the setup function here rather than in the test file itself? I was using `helpers.ts` as a place to put helper functions used by all tests - could move a lot of it to sx.js at somepoint too

Reply:
Hmm I can see this `setup` function being used in a couple of different tests (and actually I believe it's already the case), that's why I put it here in the helpers.ts. I could however make a `setup.ts` rather than putting it here in the `helper.ts`. wdyt?

Reply:
I kind of prefer it in the test file (even if it gets duplicated) as i see it as part of the test rather than just a helper function.

Reply:
actually scrap that, on second thoughts having a `setup.ts` file with the various setup functions in would make more sense. Then  we have `vanillaSetup()` etc..

Reply:
And we keep `helpers.ts` for functions that dont use hardhat - we we should be able to move to sx.js soon

Reply:
[b801cf1](https://github.com/snapshot-labs/sx-core/pull/67/commits/b801cf19de7c8c57b309d2260b04a571153cea13)